modules/apps/portal-workflow/portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/internal/runtime/integration/test/WorkflowDefinitionManagerTest.java

```diff
@@ -133,6 +136,28 @@ public void testDeployWorkflowDraftDefinition() throws Exception {
 			workflowDefinition.getName(), deployedWorkflowDefinition.getName());
 	}
 
+	@Test
+	public void testGetWorkflowDefinitionByExternalReferenceCodeMBModeration()
+		throws Exception {
+
+		String content = StringUtil.read(
+			getResourceInputStream(
+				"message-boards-moderation-workflow-definition.xml"));
+
+		WorkflowDefinition workflowDefinition =
+			_workflowDefinitionManager.deployWorkflowDefinition(
+				"L_MESSAGE_BOARDS_USER_STATS_MODERATION",
+				TestPropsValues.getCompanyId(), TestPropsValues.getUserId(),
+				StringPool.BLANK, "message-boards-user-stats-moderation",
+				content.getBytes());
```

Comment:
@mxs2 can we use the method `_workflowDefinitionManager.getWorkflowDefinitionByExternalReferenceCode`? if so use it by passing both ERCs